server/webapp/WEB-INF/rails.new/app/views/admin/pipeline_groups/index.html.erb

```diff
@@ -46,20 +46,33 @@
                     <tbody>
                     <% group.each do |pipeline| %>
                         <tr class="pipeline">
-                            <td class="name"><%= link_to pipeline.name(), pipeline_edit_path(:pipeline_name => pipeline.name(), :current_tab => "general"), :class => 'wrapped_word' %></td>
+                            <td class="name">
+															<% if pipeline.isLocal() %>
+															  <%= link_to pipeline.name(), pipeline_edit_path(:pipeline_name => pipeline.name(), :current_tab => "general"), :class => 'wrapped_word' %>
+															<% else %>
+																<%= pipeline.name() %>
```

Comment:
Please add `:class => 'wrapped_word'` for this too just in case pipeline name is too long. Also, the formatting a bit off :)